src/pact-util.ts

```diff
@@ -10,8 +11,8 @@ const checkTypes = require("check-types");
 export const DEFAULT_ARG = "DEFAULT";
 
 export class PactUtil {
-	public get cwd():string {
-		return path.resolve(__dirname, "..");
+	public get binaryBasePath():string {
+		return path.join(os.tmpdir(), "pact");
```

Comment:
On Mac/Linux, won't this result in the pact binary mysteriously "disappearing"? Files in `$TMPDIR` will be cleaned up occasionally by the OS on a Mac (default every 3 days) and on a reboot on Linux.